karma.base.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  basePath: "",
+  frameworks: ["webpack", "mocha", "chai"],
+  files: ["test/unit/**/!(*.node).test.ts"],
+  exclude: [],
+  preprocessors: {
+    "test/**/*.ts": ["webpack"],
+  },
+  reporters: ["spec"],
+  browsers: ["ChromeHeadlessCI", "Electron", "FirefoxHeadless"],
+  customLaunchers: {
+    ChromeHeadlessCI: {
+      base: "ChromeHeadless",
+      flags: ["--no-sandbox"],
```

Comment:
Why `--no-sandbox`?

Reply:
I was testing some error. Removed it now.